guides/welfare/raising_an_issue.md

```diff
@@ -6,17 +6,11 @@ From time to time people may have problems or concerns about their colleagues, c
 
 The first thing you can do, if you feel comfortable and able, is raise the issue directly with the person involved. This type of direct and informal approach is encouraged and we expect your colleagues who are approached responsibly to respond in a similar manner. This approach may be particularly appropriate to dealing with matters of speech and expression, where individuals may not be aware that their behaviour is causing distress.
 
-## Raise feedback via Continuous Feedback
+## Raise the issue via your mentor
 
-If you feel unable to have a conversation directly with the person involved, and they are a member of staff at Made Tech then you can encourage them to discuss the issue in their next [continuous feedback](../../team-norms/continuous_feedback.md) session by raising the feedback through that process.
+If you feel unable to have a conversation directly with the person involved, and they are a member of staff at Made Tech, then you can discuss this further with your mentor. They can help you find a way to discuss this directly with the person or discuss other routes to help resolve the issue.
```

Comment:
Should we be referring to "mentor" or "line manager" here?

Reply:
I actually don't want to use the word line manager but haven't aligned on this with the rest of the team. Just because to be a line manager sounds like a very traditional way to work and doesn't reflect what we mean by it. Maybe I need to have these conversations first though :D

Reply:
I don't love the phrase line manager either, just wanted to make sure this wasn't accidental.

I want someone reading this to not be confused/worried, as they could be in a difficult position when they're reading it.